hexapod_dart/src/hexapod_dart_simu.cpp

```diff
@@ -334,3 +291,86 @@ void HexapodDARTSimu::_add_floor()
 
     _world->addSkeleton(floor);
 }
+
+double HexapodDARTSimu::_min_dist_angle(double a1, double a2)
```

Comment:
Because I want the minimum signed difference. Not just the difference. It does not work properly only with `res = x-y` and then loops to ensure bounds.